aws-postgresql.yml

```diff
@@ -295,6 +304,8 @@ provision:
     default: ${maintenance_day}:${maintenance_start_hour}:${maintenance_start_min}-${maintenance_day}:${maintenance_end_hour}:${maintenance_end_min}
     overwrite: true
     type: string
+  - name: backup_window
+    default: ${backup_day}:${backup_start_hour}:${backup_start_min}-${backup_day}:${backup_end_hour}:${backup_end_min}
```

Comment:
issue: Automated backups occur **daily** during the preferred backup window (UTC). 

The correct format would be `03:00-04:00`. I would simplify the properties, eliminating all of them and using only one type string. I'd also add this [link](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithAutomatedBackups.html) as a reference to the customer.